core/amber/src/main/resources/texera-compiler-web-config.yml

```diff
@@ -0,0 +1,42 @@
+server:
+  # modify applicationContextPath if you want the root path to be the name of the application
+  # for example, set it to /twitter, then the url will become texera.ics.uci.edu:port/twitter
+  applicationContextPath: /
+  applicationConnectors:
+    - type: http
+      port: 9090
+  adminConnectors:
+    - type: http
+      port: 9091
+  requestLog:
+    type: classic
+    timeZone: UTC
+    appenders:
+      - type: console
+      - type: file
+        currentLogFilename: ../log/access-texera-compiler.log
+        threshold: ALL
+        queueSize: 512
+        discardingThreshold: 0
+        archive: true
+        archivedLogFilenamePattern: ../log/access-texera-compiler-%d{yyyy-MM-dd}.log.gz
+        archivedFileCount: 7
+        bufferSize: 8KiB
+        immediateFlush: true
```

Comment:
from now on, I think we can combine access logs with the dropwizard log. otherwise each service we will have two log files, which is not convenient when debugging.

Reply:
Done.